earth_enterprise/BUILD_RPMS.md

```diff
@@ -0,0 +1,16 @@
+# Installing via Package Managers
+This document describes how to build and install Open GEE packages (i.e. RPMs, and later, DEBs).  Packages must be built on the same platform that they are installed on to avoid library errors.  This document assumes you have development system of the target platform set up and ready to build Open GEE.
+
+Currently only RPMs are supported.  You start with a normal Open GEE build environment
+that you can use for development on Centos, RHEL, or Ubuntu machines via the existing [install scripts](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server).
```

Comment:
I'm a bit confused by this sentence.  What is it trying to say?  If we're saying that we start with a normal build environment the link should probably be to the build instructions rather than the install instructions.

I think when I started on this sentence I was trying to indicate that RPMs don't work for Ubuntu so you still have to use the install scripts.

Reply:
Yes, that should be pointing to the build scripts...